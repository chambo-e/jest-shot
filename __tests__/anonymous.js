import shot from '../src/';
import AnonymousComponent from '../__components__/anonymous';

shot(AnonymousComponent);
