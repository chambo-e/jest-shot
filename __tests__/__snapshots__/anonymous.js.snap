// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unknown renders correctly without props 1`] = `
<p>
  Hello
</p>
`;
